3-semantic-html/basic.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta name="viewport" content="width=device-width,initial-scale=1.0">
+        <meta charset="utf-8">
+        <title>Ferreteria Andres Rivera</title>
+    </head>
+    <body>
+        <header>
+        <h1>FERRETERIA</h1>
+        <h2>Bienvenidos a la Ferreteria la 14</h2>
+        <nav>
+            <ul>
+                <li>Inicio</li>
+                <li>Pituras</li>
+                <li>Herramientas</li>
+                <li>Tuberia</li>
+                <li>Accesorios</li>
+            </ul>
+        </nav>
+        </header>
+        <main>
+            <aside>
+                <p>A continuacion encontraras nuestro productos que en el momento estan en promocio.</p>
+            </aside>
+            <article>
+                <header>
+                    <h1>PROMOCIONES DE MOMENTO</h1>
+                    <img src="../3-semantic-html/imagenes/productosFerreteria.jpg" width="250" height="250"  alt="Productos de ferreteria">
+                    <br>
+                    <figcaption>
+                        Productos de ferreteria
+                    </figcaption>
+                </header>
+            <section>
+                <p>Esta ferreteria es un establecimiento comercial que se dedica  a la venta al por mayor de materiales para construccion y a las necesidades del hogar.</p>  
+                <img src="../3-semantic-html/imagenes/ferreteria.jpg" width="700" height="300" alt="Fachada">
+            </section>
+                <footer>
+                    <nav>
+                        <ol>
```

Comment:
```suggestion
                        <ul>
```
It's not common to use the ordered list in the footer.